jaegertracing-kubernetes-deployment-itest/src/main/java/io/jaegertracing/kubernetes/deployment/BaseETest.java

```diff
@@ -128,6 +134,14 @@ public void testDependencyLinks() throws IOException, InterruptedException {
     }
   }
 
+  @Test
+  public void hitDependencyScreen() throws IOException {
+    InputStream response = new URL(queryUrl + "/api/dependencies?endTs=0").openStream();
```

Comment:
Could you use the same http client as in other tests? There is probably missing an assert on the status code.